css-pseudo-4/Overview.bs

```diff
@@ -49,6 +49,11 @@ spec:fill-stroke-3; type:property; text:stroke-width
   and on their syntax and interaction with other <a>selectors</a>,
   see [[!SELECTORS-4]].
 
+<h2 id="styling">Styling Pseudo-elements</h2>
+
+  [=Custom properties=] [[!CSS-VARIABLES-1]] are always allowed in pseudo-element styles,
+  even if the pseudo-element has a restricted set of CSS properties applying to it.
+
```

Comment:
I think it may be better to explicitly list custom properties in the various lists of allowed properties for pseudo-elements.

And also in https://drafts.csswg.org/css-variables-1/#defining-variables
> Applies to: all elements (Includes ::before and ::after pseudo-elements.)

should probably clarify that all pseudo-elements are included, not just ::before and ::after

Reply:
i’m happy to move this to the local lists only, if that’s what you mean. note that in the resolution for #6264, @tabatkins seemed to prefer the blanket statement:

> \<bkardell_> TabAtkins: we should probably make a blanket statement that they are valid on all pseudo elements

Reply:
How about this rephrasing?
```suggestion
<h2 id="styling">Styling Pseudo-elements</h2>

  [=Custom properties=] [[!CSS-VARIABLES-1]] can be specified on--
  and inherited into--
  pseudo-elements,
  even if they do not apply
  due to the pseudo-element having a restricted set of applicable CSS properties.
```

I think I like it a little better, because in the original "allowed" is a bit vague. Allowed to do what?

Reply:
(thanks for the feedback (here and #6665) @frivoal, i’m on leave right now, but will take a look after 4 january)

Reply:
done in 5b50dee956cb9faec57258bd1e68ddf9e2d9d20f with a minor tweak to “do not apply”, what do you think?